open_spiel/python/mfg/algorithms/best_response_value.py

```diff
@@ -82,13 +82,16 @@ def eval_state(self, state):
       self._state_value[state_str] = (
           state.rewards()[0] + self.eval_state(new_state))
       return self._state_value[state_str]
-    else:
-      assert state.current_player() == 0, "The player id should be 0"
+    assert state.current_player() == pyspiel.PlayerId.DEFAULT_PLAYER_ID, (
```

Comment:
(Not true anymore with multi-pop, we'll try to push the support for it so that you can sync).

Reply:
Thanks